src/app/rpmostree-builtin-start-daemon.cxx

```diff
@@ -45,7 +45,7 @@ typedef enum {
 
 static AppState appstate = APPSTATE_STARTING;
 static gboolean opt_debug = FALSE;
-static char *opt_sysroot = "/";
+static char *opt_sysroot = NULL;
```

Comment:
Huh interesting, why did `g++` complain about this?

Reply:
Because `char *` is mutable and static strings aren't.